add some new phrases & reorder story so it makes more sense

LornitaR opened the pull request.

LornitaR committed:
add some new phrases
reorder story so it makes more sense

LornitaR committed:
add new phrases

kieranwhooley commented on src/data/cabhair.js:
```diff
@@ -1,15 +1,18 @@
+            "en": "After a while, the woman called my parents. "
+        }, {
+            "ga": "Tháinig an otharcharr ar nós na gaoithe. ",
+            "en": "After a while, The ambulance arrived like the wind. "
```
Needs a lowercase t on the first The in 'After a while, The ambulance arrived like the wind. ' :)

LornitaR replied:
I just removed the `After a while` bit, since we didn't want to say that as well as saying the ambulance arrived really quickly.

kieranwhooley reviewed (changes requested):
Minor change in the case of the text needed in the cabhair.js file

LornitaR committed:
Update cabhair.js

I removed the "after a while", because we'd decided we didn't need that before this sentence.
